include/generic/SugarWidgets/SugarWidgetSubPanelDetailViewLink.php

```diff
@@ -121,9 +121,17 @@ public function displayList(&$layout_def)
             if ($module == 'EAPM') {
                 $link = "index.php?module=$module&action=$action&record={$record}{$parent}";
             }
-            return '<a href="' . $link . '" >'."$value</a>";
-        } else {
-            return $value;
-        }
-    }
+            if( $module == 'DocumentRevisions'){
```

Comment:
Wrong indentation in the additional code. Please use PSR standards.
(Also this function complexity getting too much, needs to split smaller logical units.)